src/reuse/modules/mobile/Mobile.ts

```diff
@@ -0,0 +1,14 @@
+import gestures, { Gestures } from "./gestures";
+import element, { ElementModule } from "./element";
+import userInteraction, { UserInteraction } from "./userInteraction";
+import device, { Device } from "./device";
+
+export class Mobile {
+  gestures: Gestures = gestures;
+  userInteraction: UserInteraction = userInteraction;
```

Comment:
Is there a significant difference between actions in the `gestures`and `userInteraction`namespaces? I am not a mobile tester, just curious if it's really better to have these two different namespaces instead of one

Reply:
Mobile gestures and user interactions are closely related but not the same. Mobile gestures refer specifically to touch-based commands, like tapping, swiping, or pinching, that users perform on a device to execute actions. User interactions encompass a broader range of ways users engage with the app, including gestures as well as other forms of input like button clicks or voice commands. 

Essentially, gestures are a subset of user interactions.

Refer to below article for more info:
https://www.justinmind.com/blog/app-interaction-gestures/#:~:text=interactions%20in%20Justinmind-,Difference%20between%20app%20interactions%20and%20gestures,users%20gesticulate%20and%20apps%20interact.